### ISSUE TITLE
is there some pictures for comparision?

### ISSUE THREAD

**Issue by zzzgit** [geowarin/friendly-errors-webpack-plugin#29]
Hello
What are the differences between the result when before using this plugin and the result when after that?
Are there some pics?

**Comment by geowarin** [comment]
Hello !

As you can see, there isn't such a comparison (yet!).
The plugin is really easy to set up, I suggest you try with a few use cases (ts-lint, babel error and modules not found) and see if you like it.

If you have some time to make some screenshots I'll gladly put them in the doc :smile:

**Comment by zzzgit** [comment]
Yes,I tried "modules not found" as you mentioned,then I found that if you haven't use this plugin, there will be no error in the console when modules not found.
The default behavior is doing nothing,It's odd!
